### ISSUE TITLE
Snippet to be applied to multiple scopes - Ex. Sass and CSS

### ISSUE THREAD

**Issue by caryevans** [chuyik/brackets-snippets#72]
Awesome extension!
It would be dandy if a snippet name could support more than one scope (file type)

Ex: "red" - I would like it to generate a red border (border:1px solid red;) that I use in HTML, CSS and LESS/SASS for debugging.

It's entirely possible that I'm missing something, but for some reason, it does not allow me to use the same keyword and assign it multiple "scopes".

**Comment by Lomacar** [comment]
YES PLEASE! Or at the very least make it so snippets in CSS also work automatically in LESS and SASS. It is crazy not to have that.

**Comment by jdpoenisch** [comment]
As workarround you could just create an equal snippet for every scope you want it.

**Comment by Lomacar** [comment]
Obviously that is what we want to avoid doing.

**Comment by jdpoenisch** [comment]
well the only alternative I see so far would be to set the scope to "--any--" so at least you don't have to configure them multiple times
